packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<header>
+	<header class="border-b" :style="{ borderColor: 'var(--surface-border-alt)' }">
```

Comment:
You can just change line 46 to have a different border-color

Reply:
Oh. Right. :) I'll do that now.